lib/mayaUsd/fileio/primWriterArgs.h

```diff
@@ -41,8 +41,7 @@ class UsdMayaPrimWriterArgs
     UsdMayaPrimWriterArgs(
         const MDagPath&     dagPath,
         const bool          exportRefsAsInstanceable,
-        const TfToken::Set& excludeExportTypes,
-        const std::string&  defaultPrim);
```

Comment:
The default prim was not used anywhere so I removed it.

Reply:
I also checked and the _defaultPrim is private and isn't being used from anywhere. Good catch!